apps/web/src/components/rich-editor.tsx

```diff
@@ -163,9 +165,21 @@ export function RichEditorToolbar(props: {
       />
 
       <Show when={props.onFileAdded !== undefined}>
-        <label role="button" class="rounded-sm p-0.5">
-          <Dynamic component={IconPaperClip} class="size-5 text-dim transition-colors hover:text-text" />
-          <input type="file" onChange={(event) => props.onFileAdded?.(event)} class="sr-only" />
+        <label
+          role="button"
+          class="rounded-sm p-0.5 aria-disabled:cursor-default"
```

Comment:
We can probably use `has-disabled:pointer-events-none` (and remove `aria-disabled`)

Same for the other one, in the comment form.